plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/AllManifestsTable.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg;
```

Comment:
Is the documenation for this table on https://trino.io/docs/current/connector/iceberg.html#metadata-tables coming as a follow-up ?

Reply:
I should .. I think @Ordinant @jhlodin and team are working on it .. I wont be able to

Reply:
But yes .. it would have been better to do the docs as part of this PR so the release notes can link to docs and users get a feature with docs at the same time